src/CommentSold/Api/Resources/Request/Category/GetSubCategoriesRequest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CommentSold\Api\Resources\Request\Category;
+
+use CommentSold\Api\Exception\InvalidArgumentException;
+use CommentSold\Api\Resources\Request\AbstractRequest;
+use CommentSold\Api\Services\AbstractService;
+
+class GetSubCategoriesRequest extends AbstractRequest
+{
+    public int $category_id = 0;
+    public int $page = 1;
+    public int $perPage = AbstractService::PER_PAGE;
```

Comment:
I don't really like that the Request is pulling from AbstractService. I think it could have it's own set default, a composed default from a more neutral area than AbstractService, or even null and just use the server's default.